src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,330 @@
+package duke;
```

Comment:
This file is really long! Might I suggest breaking it down into more classes?

Reply:
Yes, I'll clean this class up after the GUI is properly implemented. Right now I know there are a lot unnecessary lines in it.